Modules/selectmodule.c

```diff
@@ -202,10 +226,6 @@ set2list(fd_set *set, pylist fd2obj[FD_SETSIZE + 1])
     return NULL;
 }
 
-#undef SELECT_USES_HEAP
-#if FD_SETSIZE > 1024
-#define SELECT_USES_HEAP
```

Comment:
I think `SELECT_USES_HEAP` should be preserved for non-Windows platforms.
What is the reason for removing this mode?

Reply:
I started from vstinner's prototype: #13845 which removed all the places where it was used. Since it was not used anymore, I removed the definition of it too. But I can add it back if needed.